transform/mattermost-analytics/models/staging/salesforce/_salesforce__models.yml

```diff
@@ -0,0 +1,1661 @@
+
+      - name: product2_id
+        description: ""
+
+      - name: product_code
+        description: ""
+
+      - name: quantity
+        description: ""
+
+      - name: service_date
+        description: ""
+
+      - name: sort_ordr
+        description: ""
+
+      - name: subtotal
+        description: ""
+
+      - name: systemmodstamp
+        description: ""
+
+      - name: total_price
+        description: ""
+
+      - name: unit_price
+        description: ""
+
+      - name: amount_manual_override__c
+        description: ""
+
+      - name: coterm_expansion_amount__c
+        description: ""
+
+      - name: discount_calc__c
+        description: ""
+
+      - name: discounted_list_price__c
+        description: ""
+
+      - name: dwh_external_id__c
+        description: ""
+
+      - name: end_date__c
+        description: ""
+
+      - name: expansion_amount__c
+        description: ""
+
+      - name: invoice_id__c
+        description: ""
+
+      - name: invoice_status__c
+        description: ""
+
+      - name: is_prorated_expansion__c
+        description: ""
+
+      - name: leftover_expansion_amount__c
+        description: ""
+
+      - name: lineitemid__c
+        description: ""
+
+      - name: monthly_billing_amount__c
+        description: ""
+
+      - name: multi_amount__c
+        description: ""
+
+      - name: netsuite_invoice_id__c
+        description: ""
+
+      - name: new_amount__c
+        description: ""
+
+      - name: pricing_method__c
+        description: ""
+
+      - name: product_end_datef__c
+        description: ""
+
+      - name: product_line_type__c
+        description: ""
+
+      - name: product_start_datef__c
+        description: ""
+
+      - name: product_type__c
+        description: ""
+
+      - name: prorated_listprice__c
+        description: ""
+
+      - name: recalculate_sales_price__c
+        description: ""
+
+      - name: renewal_amount__c
+        description: ""
+
+      - name: renewal_end_date__c
+        description: ""
+
+      - name: renewal_multi_amount__c
+        description: ""
+
+      - name: renewal_start_date__c
+        description: ""
+
+      - name: renewed_by_opportunity_line_id__c
+        description: ""
+
+      - name: reseller_fee__c
+        description: ""
+
+      - name: revenue_type__c
+        description: ""
+
+      - name: sales_price_needs_to_be_updated__c
+        description: ""
+
+      - name: start_date__c
+        description: ""
+
+      - name: subs_id__c
+        description: ""
+
+      - name: subs_version_id__c
+        description: ""
+
+      - name: term_months__c
+        description: ""
+
+      - name: total_amount__c
+        description: ""
+
+      - name: total_amount_total_price__c
+        description: ""
+
+      - name: total_price_with_annualized_expansion__c
+        description: ""
+
+  - name: stg_salesforce__user
+    description: | 
+      A user is anyone who logs in to Salesforce.
```

Comment:
is this really a valid description? Not sure why we'd need to handle "users that log into SFDC" ??

Reply:
> is this really a valid description? Not sure why we'd need to handle "users that log into SFDC" ??

It actually is. Perhaps we can change the wording. Here's a different one: https://developer.salesforce.com/docs/atlas.en-us.object_reference.meta/object_reference/sforce_api_objects_user.htm

This model is used in the old project as part of ARR models.

Reply:
Yes, I'd think the second one is more appropriate? I would not equate "Represents a user in your organization" with "A user is anyone who logs in to Salesforce." - I assume there are plenty of MM users that never logged-in into SFDC.

Reply:
Done!